src/components/Account.js

```diff
@@ -187,88 +187,84 @@ function Account(props) {
   ));
 
   return (
-    <>
-      <Grid container className={classes.box}>
-        <Grid item xs={3}>
-          <Paper elevation={3} className={classes.menu}>
-            <Menu variant="plain" />
-          </Paper>
-        </Grid>
-        <Grid item xs={9}>
-          <Grid container className={classes.rightBox}>
-            <Grid item xs={12}>
-              <Grid container className={classes.titleBox}>
-                <Grid item>
-                  <AccountIcon className={classes.accountIcon} />
-                </Grid>
-                <Grid item>
-                  <Typography variant="h2">Account</Typography>
-                </Grid>
+    <div style={{ display: 'flex', flexDirection: 'row', flexWrap: 'nowrap' }}>
```

Comment:
Set this container to `width: '100%'` to ensure it fills the screen.
It's also better to create a class for these CSS properties – see line 23.